content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kubernetes sangat mudah dikonfigurasi dan dikembangkan. Sehingga,
+jarang membutuhkan fork atau menambahkan patches ke kode projek Kubernetes.
```

Comment:
```suggestion
jarang dibutuhkan _fork_ atau menambahkan _patch_ ke kode proyek Kubernetes.
```

Italicize English words needed to be included in the translation.

Proyek is the standard/baku form of projek. Use standard words.
https://kbbi.kemdikbud.go.id/entri/projek

Reply:
done